R/primary-keys.R

```diff
@@ -239,6 +244,7 @@ dm_get_all_pks_def_impl <- function(def, table = NULL) {
 #' @export
 #' @examplesIf rlang::is_installed("nycflights13") && rlang::is_installed("DiagrammeR")
 #' dm_nycflights13() %>%
+#'   dm_rm_fk(flights, origin, airports) %>%
 #'   dm_rm_pk(airports, fail_fk = FALSE) %>%
 #'   dm_draw()
```

Comment:
If we really must change the example:

```suggestion
#' dm_nycflights13() %>%
#'   dm_rm_fk(flights, origin, airports) %>%
#'   dm_rm_pk(airports) %>%
#'   dm_draw()
```

Reply:
sure we can do that. 
Actually, in the near future (#1729), the dm_rm_fk() call won't be necessary anymore. Shall we then keep the argument `fail_fk` for `dm_rm_uk()`/`dm_rm_pk()`? I guess if we keep it, the default would need to change to `FALSE` and all FKs would be removed then.

Reply:
I think the right course of action is to deprecate the `fail_fk` argument and just keep the foreign key pointing to a then implicit unique key.